certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -3154,6 +3154,93 @@
         "deprecated": false,
         "x-ms-no-generic-test": true,
         "x-ms-visibility": "advanced"
+      },
+      "put": {
+        "tags": [
+          "DocuSign"
+        ],
+        "summary": "Update recipient tabs values",
+        "description": "Updates one or more tabs for a recipient in a draft envelope.",
+        "operationId": "UpdateRecipientTabsValues",
+        "parameters": [
+          {
+            "name": "accountId",
+            "in": "path",
+            "description": "Account id",
+            "required": true,
+            "x-ms-summary": "Account",
+            "x-ms-test-value": "insert account id",
+            "x-ms-dynamic-values": {
+              "operationId": "GetLoginAccounts",
+              "value-collection": "loginAccounts",
+              "value-path": "accountIdGuid",
+              "value-title": "name"
+            },
+            "type": "string"
+          },
+          {
+            "name": "envelopeId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Envelope id",
+            "x-ms-summary": "Envelope",
+            "x-ms-test-value": "insert envelope id"
+          },
+          {
+            "name": "recipientId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Recipient id",
+            "x-ms-summary": "Recipient",
+            "x-ms-test-value": "insert recipient id"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "tabType": {
+                    "type": "string",
+                    "description": "Tab type",
+                    "x-ms-summary": "Tab type",
```

Comment:
Just say Type for x-ms-summary, otherwise it will show in the ui as `Tab Tab type`

Reply:
Done